tool/ruby_vm/views/_trace_instruction.erb

```diff
@@ -11,6 +11,13 @@
 INSN_ENTRY(<%= insn.name %>)
 {
     vm_trace(ec, GET_CFP());
-    DISPATCH_ORIGINAL_INSN(<%= insn.jump_destination %>);
+    <%-
+    if insn.name =~
+    /\Atrace_opt_(plus|minus|mult|div|mod|eq|neq|lt|le|gt|ge|ltlt|and|or|aref|aset|length|size|empty_p|nil_p|succ|not|regexpmatch2)\z/
+      jump_dest = "opt_send_without_block"
+    end
+    -%>
```

Comment:
```suggestion
%   if insn.name =~
%   /\Atrace_opt_(plus|minus|mult|div|mod|eq|neq|lt|le|gt|ge|ltlt|and|or|aref|aset|length|size|empty_p|nil_p|succ|not|regexpmatch2)\z/
%     jump_dest = "opt_send_without_block"
%   end
```
I'm not sure if you like this, but we use this `%` syntax in VM templates as well.